src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -364,31 +365,32 @@ function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourceLoade
         .map(assembly => resourceLoader.loadResource(assembly, `_framework/${assembly}`, lazyAssemblies[assembly], 'assembly'))
         .map(async resource => (await resource.response).arrayBuffer()));
 
+
       return BINDING.js_to_mono_obj(
         Promise.all([resourcePromises, pdbPromises]).then(values => {
-          const resourcesToLoad = values[0];
-          const pdbsToLoad = values[1];
-          if (resourcesToLoad.length) {
+          lazyResources["assemblies"] = values[0];
```

Comment:
Could you elaborate on what was the issue was and how this fixes it?

Reply:
Just added some notes on this in the description.